neofetch

```diff
@@ -1197,6 +1198,17 @@ get_kernel() {
     fi
 }
 
+get_boot_method() {
+    case "$os" in
+        "Linux")
+            [ -d /sys/firmware/efi ] && boot_method="UEFI" || boot_method="BIOS"
+        ;;
+        *)
```

Comment:
Remove this whole block. Neofetch doesn't print "Unknown" for anything, instead it silently skips displaying the information. Using `-v` will show you what couldn't be detected instead.

Reply:
Ah, nice. Will take care of these issues and then look at what is involved in supporting some of the other OS.